x/ecocredit/server/core/msg_remove_class_creators_test.go

```diff
@@ -0,0 +1,96 @@
+package core
+
+import (
+	"encoding/json"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/regen-network/gocuke"
+	"github.com/stretchr/testify/require"
+
+	ecocreditv1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1"
+	"github.com/regen-network/regen-ledger/x/ecocredit/core"
+)
+
+type removeClassClassCreator struct {
+	*baseSuite
+	err error
+}
+
+type classCreators struct {
+	Creators []string `json:"creators"`
+}
+
+func TestRemoveClassCreators(t *testing.T) {
+	gocuke.NewRunner(t, &removeClassClassCreator{}).Path("./features/msg_remove_class_creator.feature").Run()
+}
+
+func (s *removeClassClassCreator) Before(t gocuke.TestingT) {
+	s.baseSuite = setupBase(t)
+}
+
+func (s *removeClassClassCreator) ClassCreatorsWithProperties(a gocuke.DocString) {
+	var creators classCreators
+
+	err := json.Unmarshal([]byte(a.Content), &creators)
+	require.NoError(s.t, err)
+
+	for _, creator := range creators.Creators {
+		creatorAddr, err := sdk.AccAddressFromBech32(creator)
+		require.NoError(s.t, err)
+
+		err = s.stateStore.AllowedClassCreatorTable().Save(s.ctx, &ecocreditv1.AllowedClassCreator{
```

Comment:
We should be able to use `Insert` here rather than `Save` and generally we should favor `Insert` over `Save` to catch any mistakes in writing tests or unexpected behavior in the implementation.

```suggestion
		err = s.stateStore.AllowedClassCreatorTable().Insert(s.ctx, &ecocreditv1.AllowedClassCreator{
```